DigitalLearningSolutions.Data/Services/CourseAdminFieldsService.cs

```diff
@@ -176,5 +212,26 @@ DelegateCourseInfo delegateCourseInfo
 
             return list;
         }
+
+        private static CustomPrompt? PopulateCustomPrompt(
```

Comment:
We shouldn't have these static methods that are common to both services here, we should be using the ones in the separate file